### PR TITLE
add CCFileUtils Lua support

### PR DESCRIPTION
**Pull request by dualface**

### CODE REVIEW

File: cocos2dx/platform/CCFileUtils.h [hunk 1]
```diff
@@ -210,6 +210,14 @@ class CC_DLL CCFileUtils : public TypeInfo
      *  @since v2.1
      */
     virtual void setSearchResolutionsOrder(const std::vector<std::string>& searchResolutionsOrder);
+
+    /**
+      * Append search order of the resources.
```
Comment (dumganhar) [inline cocos2dx/platform/CCFileUtils.h:215]: `Append search order of the resources` -> `Adds a search order of the resources` ?

File: cocos2dx/platform/CCFileUtils.h [hunk 2]
```diff
@@ -239,6 +247,13 @@ class CC_DLL CCFileUtils : public TypeInfo
     virtual void setSearchPaths(const std::vector<std::string>& searchPaths);
     
     /**
+      * Add search path.
```
Comment (dumganhar) [inline cocos2dx/platform/CCFileUtils.h:250]: Adds a search path.
@param path The search path to be added.

File: cocos2dx/platform/CCFileUtils.h (hunk 1 above)
Comment (dualface) [inline cocos2dx/platform/CCFileUtils.h:215]: addSearch... for lua

ÔÚ 2013-2-4£¬13:55£¬James Chen notifications@github.com Ð´µÀ£º

**Comment by dumganhar** [comment]
Thanks.

**Pull request merged**